CHANGELOG

```diff
@@ -1,4 +1,20 @@
 Unreleased
+	Support full committee options in OpenAPI3
+	Fix bug when non defined link with form content type and coerce option (#173)
+	Support check_content_type option for OpenAPI3 (#174)
+
+3.0.0.alpha 2018-12-26
+	OpenAPI 3.0 support (#164)
+	Drop old ruby version (#146)
+
+2.4.0 xxxxx
+	Add methods for harmless upgrade (#199, #195)
+	Support filepath option (#191)
+	Change change validate_errors option to validate_success_only (#187)
+	Bugfix for coerce_recursive option in request params (#162)
+	Path definition fewer matches in OpenAPI2 (#160)
+	Deprecated test method (#157)
+	Add `error_handler` option (#152)
```

Comment:
Sorry, I rewrote some of the 2.4.0 CHANGELOG entries in 114edc73bc433814fd25d2dea10b4476b48500db so you may have to rebase. I would just replace this whole block with the new set.